### PR TITLE
VORTEX-6042: Feature view quick picker window

### PR DESCRIPTION
**Pull request by rkausch**
Fixes #6042

## Proposed Changes
  - Add a little window (accessible from the tools menu) to quickly jump to saved view areas, usually during presentations using MIST as the visualization area.

### CODE REVIEW

File: open-sphere-plugins/pom.xml [hunk 1]
```diff
@@ -48,5 +48,6 @@
         <module>wfs</module>
         <module>wms</module>
         <module>wps</module>
-    </modules>
-</project>
+    <module>view-quick-picker</module>
+  </modules>
+</project>
```
Comment (Xenotide) [inline open-sphere-plugins/pom.xml]: Missing end of file line.

File: open-sphere-plugins/view-quick-picker/src/main/java/io/opensphere/view/picker/model/ViewPickerModel.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
+package io.opensphere.view.picker.model;
+
+import io.opensphere.core.viewbookmark.ViewBookmark;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+
+/**   */
```
Comment (Xenotide) [inline open-sphere-plugins/view-quick-picker/src/main/java/io/opensphere/view/picker/model/ViewPickerModel.java]: Missing comment.

File: open-sphere-plugins/view-quick-picker/src/main/java/io/opensphere/view/picker/view/ViewPickerPanel.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        myToolbox = toolbox;
+        myModel = model;
+
+        myModel.getViews().addListener(this::viewsChanged);
+    }
+
+    /**
+     * An event handler used to react to changes in the underlying model.
+     * 
+     * @param c the change propagated from the underlying model.l
```
Comment (Xenotide) [inline open-sphere-plugins/view-quick-picker/src/main/java/io/opensphere/view/picker/view/ViewPickerPanel.java]: Extraneous letter at the end + maybe a better name for the parameter?

Comment (rkausch) [reply open-sphere-plugins/view-quick-picker/src/main/java/io/opensphere/view/picker/view/ViewPickerPanel.java]: fixed

File: open-sphere-plugins/view-quick-picker/src/main/java/io/opensphere/view/picker/model/ViewPickerModel.java (hunk 2 above)
Comment (rkausch) [reply open-sphere-plugins/view-quick-picker/src/main/java/io/opensphere/view/picker/model/ViewPickerModel.java]: fixed

File: open-sphere-plugins/pom.xml (hunk 1 above)
Comment (rkausch) [reply open-sphere-plugins/pom.xml]: fixed

**Pull request merged**